### ISSUE TITLE
I need to politifact & gossipcop URL

### ISSUE THREAD

**Issue by fariba07** [KaiDMML/FakeNewsNet#17]
hello
I need URL links for each row in CSV file from Gossipcop & politifact  that you start collecting data from there. how can access this pages URL? can I use id column? HOW?
this is very important for my research . thank you for your response

**Comment by Isha222** [comment]
It is available in dataset provided. The csv files contain that information.

**Comment by fariba07** [comment]
no, I mean the url for the politifact and gossipcop pages that were labeled csv files data based on those pages. I can't find this url in csv files and I need it. thank you
